test.js

```diff
@@ -1,5 +1,20 @@
 var t = "hello";
 
-if (t === "hello') {
-    console.log('ok');
+var selec = '[data-dikkemik]';
+var doofus = true;
+
+if (t == 'hello') {
+  console.log("ok");
+}
+
+if (true) {
+  var sjaakDeman = false;
+}
+
+if (true) {
+  var cat = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["cat" is defined but never used](https://www.codacy.com/app/rubstar/rvs-pr-testing/file/2473394615/issues/source?bid=3260976&fileBranchId=3261133#l15)